chrome/content/zotero/zoteroPane.js

```diff
@@ -6027,7 +6028,44 @@ var ZoteroPane = new function()
 			this.hideRetractionBanner();
 		}
 	};
-	
+
+	this.showArchitectureWarning = async function () {
+		const remindInterval = 60 * 60 * 24 * 30;
+		const isWow64 = (await Services.sysinfo.processInfo).isWow64;
```

Comment:
I adjusted this variable name. `isWow64` means specifically 32-bit on 64-bit, so `isWin64` is a little misleading, even if it doesn't matter for the context in which we're using it.